src/components/icon-tile/index.tsx

```diff
@@ -0,0 +1,46 @@
+import s from './style.module.css'
+import classNames from 'classnames'
+import { ProductSlug } from 'types/products'
+
+type ProductBrandColor =
+  | 'neutral'
+  | Exclude<Exclude<ProductSlug, 'sentinel'>, 'hcp'>
```

Comment:
I _believe_ you can simplify this "double Exclude" with a `|` like this:

```suggestion
type ProductBrandColor = 'neutral' | Exclude<ProductSlug, 'sentinel' | 'hcp'>
```